src/components/uikit/input/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { twMerge } from "tailwind-merge";
+import { useState } from "react";
+import { cn } from "~/lib/utils";
+
+type Props = React.InputHTMLAttributes<HTMLInputElement>;
+
+export const Input = ({ className }: Props) => {
+  const [pending, setPending] = useState(false);
+  return (
+    <input
+      className={cn(
+        "rounded bg-red-500 px-4 py-2 text-white hover:opacity-70 ",
+        className,
+        {
+          "bg-blue-700": pending,
+        },
+      )}
+    ></input>
+  );
+};
```

Comment:
Схожа ситуація, тут не треба `pending` стану, так як він ніколи не використовується.

Так само треба прокидати всі `props`.

```suggestion
export const Input = ({ className, ...props }: Props) => {
  return (
    <input
      {...props}
      className={cn(
        "rounded bg-red-500 px-4 py-2 text-white hover:opacity-70",
        className
      )}
    ></input>
  );
};
```